spotify/settings.py

```diff
@@ -99,7 +99,7 @@
         'NAME': 'django.contrib.auth.password_validation.NumericPasswordValidator',
     },
 ]
-
+AUTH_USER_MODEL = 'music.Playlist'
 
```

Comment:
Create a new app called `users` and extend the `AbstractUser`  class in a new `User` model

from django.contrib.auth import get_user_model

User = get_user_model()